pilot/control/payloads/eventservicemerge.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Wen Guan, wen.guan@cern.ch, 2018
+
+
+import commands
+import os
+
+from pilot.control.payloads import generic
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class Executor(generic.Executor):
+    def __init__(self, args, job, out, err):
+        super(Executor, self).__init__(args, job, out, err)
+
+    def untar_file(self, lfn, job):
+        pfn = os.path.join(job.workdir, lfn)
+        command = "tar -xf %s -C %s" % (pfn, job.workdir)
+        logger.info("Untar file: %s" % command)
+        status, output = commands.getstatusoutput(command)
```

Comment:
please use the execute() function instead of commands.getstatusoutput directly (from pilot.util.container import execute), same comment for the other usage of getstatusoutput()